src/examples/aes/lifter/Holmakefile.gen

```diff
@@ -10,9 +10,9 @@ SMLFILES   = $(patsubst %Script.sml,%Theory.sml,$(wildcard *.sml))
 TARGETS    = $(patsubst %.sml,%.uo,$(SMLFILES))
 
 HOLHEAP    = ../../../HolBA-heap
-NEWHOLHEAP = 
+NEWHOLHEAP = # TODO: Create a heap with lifted binaries "HolBAExamples-aes-LiftedBinaries-heap"
```

Comment:
not sure we want this. or maybe one heap per example?

Reply:
I thought that's what we said :sweat_smile: Don't we want to put the lifted binaries into a unique heap so that it's fast to load?

Reply:
hmm, i didn't think about this for the examples directory in src. but why not :)